lib/network/modules/LayoutEngine.js

```diff
@@ -1720,7 +1718,7 @@ class LayoutEngine {
 
   /**
    * Abstract the getting of the position of a node so we do not have to repeat the direction check all the time.
-   * @param {vis.Node} node
+   * @param {Node} node
    * @returns {number|*}
```

Comment:
Gah? Number or just plain any type? I know you didn't type this, but this is too wide.